plugins/node/opentelemetry-instrumentation-hapi/src/instrumentation.ts

```diff
@@ -57,11 +57,15 @@ export class HapiInstrumentation extends InstrumentationBase {
   protected init() {
     return new InstrumentationNodeModuleDefinition<typeof Hapi>(
       HapiComponentName,
-      ['>=17 <21'],
+      ['>=21'],
```

Comment:
I guess the desupport of hapi 17 to 20 should be mentioned somewhere.

Reply:
`>=21` is also quite the promise. Can we be sure 22, 23, 30, 50 are all supported?